package.json

```diff
@@ -1,15 +1,13 @@
 {
-  "name": "react-aria-components",
-  "version": "0.0.3",
-  "description": "",
-  "main": "index.js",
+  "private": true,
   "repository": {
     "type": "git",
     "url": "https://github.com/juanca/react-aria-components"
   },
   "scripts": {
     "start": "webpack-serve",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lerna": "lerna"
```

Comment:
Could setup separate `bootstrap`, `add`, and `publish` commands here to make it easier.